android/KMEA/app/src/main/assets/android-host.js

```diff
@@ -266,6 +266,10 @@ function oskClearKeyPreview() {
   window.location.hash = 'dismissKeyPreview-'+fragmentToggle;
 }
 
+function signalHelpBubbleDismissal() {
+  window.location.hash = 'helpBubbleDismissed';
```

Comment:
This should use the `fragmentToggle` pattern for consistency.

```suggestion
function signalHelpBubbleDismissal() {
  fragmentToggle = (fragmentToggle + 1) % 100;
  window.location.hash = 'helpBubbleDismissed-'+fragmentToggle;
```